.gitlab/choco_build.yml

```diff
@@ -27,7 +27,7 @@ windows_choco_offline_7_x64:
 # it is run only once the msi package is pushed
 windows_choco_online_7_x64:
   rules:
-    !reference [.on_deploy_stable_or_rc_tag_on_beta_repo_branch_a7]
```

Comment:
Just FYI, this change will make the publish job manual on both RC and Stable deployement.
It will also create the job on non-rc tags that will target the beta branch, for example all the custom builds will have this job created while it was not the case before